test/utils/utils.go

```diff
@@ -24,7 +24,7 @@ import (
 	"os/exec"
 	"strings"
 
-	. "github.com/onsi/ginkgo/v2" //nolint:golint,revive
+	"github.com/onsi/ginkgo/v2/dsl/core"
```

Comment:
It is an unwritten rule but, we normally import the core Kubernetes APIs as `corev1`, importing ginkgo as `core` will cause some confusion. Maybe you can alias it as `ginkgocore`? I leave it up to you